SDWebImage/SDImageCacheConfig.h

```diff
@@ -73,13 +73,13 @@ typedef NS_ENUM(NSUInteger, SDImageCacheConfigExpireType) {
  * Setting this to zero means that all cached files would be removed when do expiration check.
  * Defaults to 1 weak.
  */
-@property (assign, nonatomic) NSTimeInterval maxCacheAge;
+@property (assign, nonatomic) NSTimeInterval maxDiskAge;
```

Comment:
This property mean staleness interval. It express the difference value of expiration date, `age` means an absolute value. `diskStalenessInterval` for discussion.
See `stalenessInterval` of `NSManagedObjectContext`.

Reply:
This is already been used widely maybe. I rename it with only `disk` keyword to make it easy to find the replacement. No changes is needed.

https://github.com/ibireme/YYCache/blob/master/YYCache/YYDiskCache.h#L114

Reply:
And that `StalenessInterval` sounds quite not for most programmer. 
And we can find another example, HTTP Cache Control use the `max-age`  represent exact what we've used. It also have a `max-stale` value, which represent `even if the cache is marked not fresh and need to cleanup, I can extend the further time`. So I think it's enough.

https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Cache-Control